src/App.js

```diff
@@ -1,24 +1,12 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import FirestoreTest from './components/FirestoreTest';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <FirestoreTest />
```

Comment:
I would imagine we wouldn't want this test component moving forward, so my instinct would be to remove it rather than add it to the routes. We only included it for testing purposes. What do you think?

Reply:
I really like being able to see working iterations of code. It makes verifying changes and accepting stories easier. But we are still in the early phase where it is easy to step on each other. If you want to keep it you really just need to coordinate with your teammates. If you merge this in to main before the other one they will have to merge your changes and ultimately decide to what to do with it, and I think it is best to discuss that beforehand.